spec/fortnox/api/base_spec.rb

```diff
@@ -96,15 +96,15 @@
       )
     end
 
-    it 'works', focus: true do
+    it 'works' do
       api = Fortnox::API::Base.new
       response1 = api.get( '/test', { body: '' })
       response2 = api.get( '/test', { body: '' })
       response3 = api.get( '/test', { body: '' })
 
-      expect( response1.parsed_response ).to_not eq( response2.parsed_response )
-      expect( response3.parsed_response ).to_not eq( response2.parsed_response )
-      expect( response1.parsed_response ).to eq( response3.parsed_response )
+      expect( response1 ).to_not eq( response2 )
+      expect( response3 ).to_not eq( response2 )
```

Comment:
Hmm, are you sure this works? I had it like that first but the test errored out even when the printed objects in the fail message were identical...

But if it works it's better this way.